docs/reference/plots/heatmap.md

```diff
@@ -116,3 +116,25 @@ Colorbar(fig[:, end+1], colorrange = joint_limits)  # equivalent
 fig
 ```
 \end{examplefigure}
+
+
+### Using a custom colorscale
+
+One can define a custom (color)scale using the `ReversibleScale` type. When the transformation is simple enough (`log`, `sqrt`, ...), the inverse transform is automatically deduced.
+
+\begin{examplefigure}{}
+```julia
+using CairoMakie
+CairoMakie.activate!() # hide
+
+x = 10.0.^(1:0.1:4)
+y = 1.0:0.1:5.0
+z = broadcast((x, y) -> x, x, y')
+
+scale = ReversibleScale(log10)
+fig, ax, hm = heatmap(x, y, z; colorscale = scale, axis = (; xscale = scale))
```

Comment:
Yeah but then I don't think the example is good, because why would you wrap `log10` if you can pass it on its own (like shown in other places in the docs as well). We don't want to teach the complicated version if it's not better, no?

Reply:
Alright, I've tweaked the example.